package.js

```diff
@@ -11,7 +11,7 @@ Package.onUse(function (api) {
   api.versionsFrom(['2.3', '2.8.1', '3.0-beta.0'])
 
   api.use([
-    'mongo',
+    'mongo@2.0.0-beta300.0',
```

Comment:
On the other packages I left it blank and I got the error that I need to specifiy a version for it

Reply:
We can't have this in a release as it would break backward compatibility.
Also for forward compatibility we need the `ddp` package listed here I think.